tests/test_utils.py

```diff
@@ -1,3 +1,4 @@
+import pytest
```

Comment:
Is this needed?

Reply:
No 🙈 was just scared that I'd break something if I removed it 🙈. Removed now.